stablehlo/transforms/VhloToVersion.cpp

```diff
@@ -77,6 +78,69 @@ FailureOr<Version> validateTargetVersion(llvm::StringRef versionRef,
   return targetVersion;
 }
 
+template <typename AttrOrType>
+bool isDialect(AttrOrType at, llvm::StringRef dialectName) {
+  return (at.getDialect().getNamespace() == dialectName);
+}
+
+LogicalResult isLegalAttribute(Operation* op, NamedAttribute const& attr,
```

Comment:
Let's remove the unused "op" parameter. (Same below).

Reply:
Ah, didn't realize that went stale. Used to use `op->emitError` but this is called during legalization so the intermediate state may be illegal but end result it fine, so we don't want to emit error.